src/templates/promotions/discounts/_edit.html

```diff
@@ -185,19 +185,38 @@
                 </div>
             {% endblock %}
         {% endembed %}
+        
+        {% set usersInput %}
+            <div class="flex">
+                <div>
+                    {{ forms.select({
+                        id: 'user-condition',
+                        name: 'userCondition',
+                        options: discount.getUserConditions(),
+                        value: discount.userCondition
+                    }) }}
+                </div>
+        
+                <div class="flex-grow{% if discount.userCondition is null or discount.userCondition == 'usersAnyOrNone' %} hidden{% endif %}">
+                    {{ forms.multiselectField({
+                        id: 'groups',
+                        name: 'groups',
+                        options: groups,
+                        values: discount.getUserGroupIds(),
+                        errors: discount.getErrors('groups'),
+                        class: 'selectize fullwidth',
+                    }) }}
+                </div>
+            </div>
+        {% endset %}
 
-        {% if groups|length %}
-            {{ forms.multiselectField({
-                label: 'User Groups'|t('commerce'),
-                instructions: "Groups for which this discount will be applicable to. Leave blank for all groups"|t('commerce'),
-                id: 'groups',
-                name: 'groups',
-                options: groups,
-                values: discount.getUserGroupIds(),
-                errors: discount.getErrors('groups'),
-                class: 'selectize fullwidth',
-            }) }}
-        {% endif %}
+        {{ forms.field({
+            id: 'user-condition',
+            label: 'User Condition'|t('commerce'),
```

Comment:
Would this make more sense as “User Group Condition” rather than “User Condition”, since every available option relates to user _groups_ rather than individual users?